packages/circuit-ui/components/NotificationModal/NotificationModal.tsx

```diff
@@ -110,68 +83,59 @@ export const NotificationModal: ModalComponent<NotificationModalProps> = ({
   preventClose = false,
   className,
   ...props
-}) => {
-  if (process.env.NODE_ENV !== 'production' && className) {
-    throw new CircuitError(
-      'NotificationModal',
-      'Custom styles are not supported by this component. If your use case requires custom styles, please open an issue at https://github.com/sumup-oss/circuit-ui.',
-    );
-  }
```

Comment:
Why did you remove this warning?

Reply:
if the Dialog component accepts className, I don't see why the NotificationModal shouldn't (also I thought maybe this was a react-modal constraint)